recipes/xsar/meta.yaml

```
@@ -0,0 +1,93 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "xsar" %}
+{% set version = "0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/umr-lops/xsar.git
+  git_rev: v{{ version }}
```

Instead of checking out from git, how about pointing to the release tarball on GitHub as noted in the comments in the template below. We also need a sha256 to go with it:
```suggestion
  url: https://github.com/umr-lops/xsar/archive/v0.5.tar.gz
  sha256: [your sha here]
```